OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -358,6 +358,7 @@ abstract class MovePart : Activity
 		{
 			protected readonly Move Move;
 			protected readonly WPos From, To;
+			protected readonly WAngle FromToYaw;
```

Comment:
Just reuse `ToFacing`

Reply:
What do you mean by reusing ToFacing? You mean `FromToYaw` is equal to `ToFacing` ?

Reply:
yes

Reply:
Actually, nope. If you just use `ToFacing` or  `ToFacing + AngleModifier` you get this:
![bug](https://user-images.githubusercontent.com/13763394/230594827-72b5e587-24ee-4e00-a68d-4ae40b490c12.gif)

Reply:
https://github.com/OpenRA/OpenRA/pull/20794#discussion_r1160617784

Reply:
fixed

Reply:
I think you may have forgotten to push the changes 🦖

Reply:
ummm, should be pushed now, weird though.